spec.html

```diff
@@ -22034,6 +22048,7 @@ <h1>GetExportedNames ( _exportStarSet_ ) Concrete Method</h1>
           </emu-alg>
           <emu-note>
             <p>GetExportedNames does not filter out or throw an exception for names that have ambiguous star export bindings.</p>
+            <p>A *null* return from GetExportedNames indicates a non-Source-Text Module Record does not support star exports. This throws a corresponding SyntaxError exception.</p>
```

Comment:
Nit: This might be clearer as a separate note

Reply:
Sure, thanks.